Sources/MEWwalletTweetNacl/TweetNacl.swift

```diff
@@ -115,4 +124,27 @@ public class TweetNacl {
     
     return Data(m[Constants.SecretBox.zeroLength..<c.count])
   }
+    
+  public static func box(message: String, nonce: Data, theirPublicKey: Data, mySecretKey: Data) throws -> Data {
+    let k = try before(publicKey: theirPublicKey, secretKey: mySecretKey)
+    guard let data = message.data(using: .utf8) else { throw TweetNaclError.tweetNacl("Invalid message")}
+    return try secretbox(message: data, nonce: nonce, key: k)
+  }
+
+  private static func secretbox(message: Data, nonce: Data, key: Data) throws -> Data {
+    guard key.count == Constants.SecretBox.keyLength else { throw TweetNaclError.invalidKey }
+    guard nonce.count == Constants.SecretBox.nonceLength else { throw TweetNaclError.invalidNonce }
+      
+    var mData = Data(count: message.count + Constants.SecretBox.boxZeroLength)
+    mData.replaceSubrange(Constants.SecretBox.boxZeroLength..<mData.count, with: message)
+    let m = [UInt8](mData)
+    var c = [UInt8](repeating: 0, count: message.count)
```

Comment:
`mData.count`, not `message.count`

Reply:
This is the declaration of mData, so it doesn't have a length yet. Should we convert `message` to `utf8 Data` and use `messageData.count`?